# msg file for interfacing with local path planner. Not sure what this entails at the moment

Header header
float32 place_holder
